src/controllers/userController.ts

```diff
@@ -0,0 +1,76 @@
+import prisma from "../../DB/dbConfig";
+import type { User } from "../../DB/dbConfig";
+import { StatusCodes } from "http-status-codes";
+import type { Request, Response } from "express";
+import bcrypt from "bcrypt";
+import { setUser } from "../utils/authJwt";
+const saltRounds = 10;
+
+interface ReqBody {
+  email: string;
+  password: string;
+}
+async function handleUserRegister(req: Request, res: Response): Promise<void> {
+  try {
+    const { email, password }: ReqBody = req.body;
+    const user: User | null = await prisma.user.findUnique({
+      where: {
+        email,
+      },
+    });
+    if (user != null) {
+      res.status(StatusCodes.CONFLICT).json({
```

Comment:
You can directly return the response like this: _return res.status().json()_
But again, it may vary on developer's way of coding.

Reply:
If I do return res.json() typescript requires the return type
What should I define as return type if I do so?